package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "notificationlib",
+    "version": "1.0.0",
+    "description": "User interface for the OCTRI Notification Library",
+    "main": "index.js",
+    "scripts": {
+        "analyze": "webpack --mode=production; webpack --profile --json > bundle-stats.json; webpack-bundle-analyzer bundle-stats.json ./target/classes/static/assets",
+        "build": "webpack --mode=production",
+        "build-css": "sass  --load-path ./node_modules/ src/main/resources/frontend/scss/app.scss ./target/classes/static/assets/css/app.css",
+        "build:dev": "webpack --mode=development",
+        "format": "prettier --write ./*.js src/main/resources/**/*.{js,vue,spec.js}",
+        "format:check": "prettier --check ./*.js src/main/resources/**/*.{js,vue,spec.js}",
+        "lint:js": "eslint ./*.js src/main/resources/**/*.{js,vue,spec.js}",
+        "start": "webpack --mode=development --progress --colors --watch",
+        "test": "karma start --single-run",
+        "test-jenkins": "karma start karma.conf.js --single-run --reporters junit --browsers Chrome_Headless"
```

Comment:
I would consider removing the test infrastructure, since we don't have any JS tests.